### PR TITLE
Removed ByteSpan completly

### PR DESCRIPTION
**Pull request by KrzysztofCwalina**
I also built new packages and uploaded them to myget

### CODE REVIEW

File: src/System.Buffers/src/System/ByteSpan.cs [hunk 1]
```diff
@@ -10,190 +10,190 @@
...
-        public byte[] CreateArray()
-        {
-            if(_length == 0) {
-                return new byte[0];
-            }
-            byte[] array = new byte[_length];
-            Marshal.Copy((IntPtr)UnsafeBuffer, array, 0, _length);
-            return array;
-        }
+    //public unsafe struct ByteSpan : IEquatable<ByteSpan>, IEnumerable<byte> {
```
Comment (krwq) [inline src/System.Buffers/src/System/ByteSpan.cs]: should the file be removed?

Comment (KrzysztofCwalina) [inline src/System.Buffers/src/System/ByteSpan.cs]: removed now. thanks for catching this.

File: demos/LowAllocationWebServer.Tests/app.config [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="System.Text.Utf8" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0" />
```
Comment (davidfowl) [inline demos/LowAllocationWebServer.Tests/app.config]: omg

**Comment by krwq** [comment]
LGTM :+1:

**Comment by livarcocc** [comment]
Looks good.

**Comment by krwq** [comment]
before we get any merge conflicts I'm merging this

**Pull request merged**